### PR TITLE
First draft of the charter

### PR DESCRIPTION
**Pull request by emersonknapp**
This fills in the template portions of the Working Group governance model, including the charter, scope, and list of projects, and some other book-keeping information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,14 +16,44 @@ Its meetings and membership are largely focused on the direction, design, and wo
...
+    * https://github.com/RobotWebTools/community
+    * https://github.com/RobotWebTools/robotwebtools.github.io
+    * https://github.com/RobotWebTools/starter-template
+    * https://github.com/RobotWebTools/rwt_dockers
+    * https://github.com/RobotWebTools/robot_web_tools
+* `roslibjs`
+  * Description: ROS 1 Javascript language client - for use in the browser
+  * Repositories:
+    * https://github.com/RobotWebTools/roslibjs
+    * https://github.com/RobotWebTools/roslibjs_integration_tests
```
Comment (emersonknapp) [inline README.md:31]: This was last updated 2014 - it seems dead. Perhaps we should remove it

File: README.md [hunk 2]
```diff
@@ -24,14 +16,44 @@ Its meetings and membership are largely focused on the direction, design, and wo
...
+    * https://github.com/RobotWebTools/ros2djs
+    * https://github.com/RobotWebTools/interactive_marker_proxy
+* Specific Data Streaming
+  * Description: Libraries for streaming ROS data effectively
+  * Repositories:
+    * https://github.com/RobotWebTools/web_video_server
+    * https://github.com/RobotWebTools/tf2_web_republisher
+    * https://github.com/RobotWebTools/webrtc_ros
+    * https://github.com/RobotWebTools/ros_web_video
+    * https://github.com/RobotWebTools/mjpeg_server
```
Comment (emersonknapp) [inline README.md:55]: This is deprecated: perhaps we should remove it or at least archive the repository

File: README.md [hunk 3]
```diff
@@ -24,14 +16,44 @@ Its meetings and membership are largely focused on the direction, design, and wo
...
+    * https://github.com/RobotWebTools/ros3djs
+    * https://github.com/RobotWebTools/ros2djs
+    * https://github.com/RobotWebTools/interactive_marker_proxy
+* Specific Data Streaming
+  * Description: Libraries for streaming ROS data effectively
+  * Repositories:
+    * https://github.com/RobotWebTools/web_video_server
+    * https://github.com/RobotWebTools/tf2_web_republisher
+    * https://github.com/RobotWebTools/webrtc_ros
+    * https://github.com/RobotWebTools/ros_web_video
```
Comment (emersonknapp) [inline README.md:54]: This is deprecated: perhaps we should remove it

File: README.md [hunk 4]
```diff
@@ -24,14 +16,44 @@ Its meetings and membership are largely focused on the direction, design, and wo
...
-* template-project
-  * Description: Brief description of project. Remove this item and add new projects using this format.
+* Organization Infrastructure
+  * Description: Repositories to manage the working group itself - these are not products for users
+  * Repositories:
+    * https://github.com/RobotWebTools/community
+    * https://github.com/RobotWebTools/robotwebtools.github.io
+    * https://github.com/RobotWebTools/starter-template
+    * https://github.com/RobotWebTools/rwt_dockers
+    * https://github.com/RobotWebTools/robot_web_tools
```
Comment (emersonknapp) [inline README.md:26]: This was last updated 2014 - it doesn't seem used. I'm not sure if it provides value? Should we remove it

File: README.md [hunk 5]
```diff
@@ -24,14 +16,44 @@ Its meetings and membership are largely focused on the direction, design, and wo
...
-
-* template-project
-  * Description: Brief description of project. Remove this item and add new projects using this format.
+* Organization Infrastructure
+  * Description: Repositories to manage the working group itself - these are not products for users
+  * Repositories:
+    * https://github.com/RobotWebTools/community
+    * https://github.com/RobotWebTools/robotwebtools.github.io
+    * https://github.com/RobotWebTools/starter-template
+    * https://github.com/RobotWebTools/rwt_dockers
```
Comment (emersonknapp) [inline README.md:25]: Is this being used for anything? Is it a demo for rwt usage?

File: README.md [hunk 6]
```diff
@@ -24,14 +16,44 @@ Its meetings and membership are largely focused on the direction, design, and wo
...
-{{
-
-* template-project
-  * Description: Brief description of project. Remove this item and add new projects using this format.
+* Organization Infrastructure
+  * Description: Repositories to manage the working group itself - these are not products for users
+  * Repositories:
+    * https://github.com/RobotWebTools/community
+    * https://github.com/RobotWebTools/robotwebtools.github.io
+    * https://github.com/RobotWebTools/starter-template
```
Comment (emersonknapp) [inline README.md:24]: Is this being kept up to date at all? Last update was 2015 - maybe it should be deprecated (or, maybe it needs to be updated)

File: README.md (hunk 1 above)
Comment (jihoonl) [reply README.md:31]: Yes. roslibjs_integration_tests can be removed.

File: README.md (hunk 2 above)
Comment (jihoonl) [reply README.md:55]: webrtc_ros and tf2_webrepublisher should be under control. ros_web_video and mjpeg_server can be archived. web_video_server is the successor.

File: README.md (hunk 4 above)
Comment (jihoonl) [reply README.md:26]: It was a sort of a meta package that includes all launchers for backend system. tf2_web_republihser, web_video_server and rosbridge. 

But, I agree to remove it. Let's keep the org. simple.

File: README.md (hunk 5 above)
Comment (jihoonl) [reply README.md:25]: It was to build a docker for demo server. But, unmaintained for a long time. It can be removed.

File: README.md (hunk 6 above)
Comment (jihoonl) [reply README.md:24]: potentially good to maintain with a robot webtool usage tutorial update. Otherwise, let's deprecate it

**Pull request merged**